libs/renterd-types/src/bus.ts

```diff
@@ -399,19 +398,19 @@ export type BucketDeleteParams = { name: string }
 export type BucketDeletePayload = void
 export type BucketDeleteResponse = void
 
-export type ObjectListParams = {
-  bucket?: string
+export type ObjectsParams = {
+  bucket: string
```

Comment:
is this now required or no?

Reply:
It's not. But if you don't provide it you get results from all buckets. So the UI should make sure to provide it.